developers/weaviate/model-providers/friendliai/generative.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Generative AI
+sidebar_position: 51
+image: og/docs/integrations/provider_integrations_friendliai.jpg
+# tags: ['model providers', 'friendliai', 'generative', 'rag']
+---
+
+# FriendliAI Generative AI with Weaviate
+
+import BetaPageNote from '../_includes/beta_pages.md';
+
+<BetaPageNote />
+
+:::info Added in `v1.27.0`
+:::
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyConnect from '!!raw-loader!../_includes/provider.connect.py';
+import TSConnect from '!!raw-loader!../_includes/provider.connect.ts';
+import PyCode from '!!raw-loader!../_includes/provider.generative.py';
+import TSCode from '!!raw-loader!../_includes/provider.generative.ts';
+
+Weaviate's integration with FriendliAI's APIs allows you to access open source and their models' capabilities directly from Weaviate.
+
+[Configure a Weaviate collection](#configure-collection) to use an FriendliAI generative AI model, and Weaviate will perform retrieval augmented generation (RAG) using the specified model and your FriendliAI API key.
+
+More specifically, Weaviate will perform a search, retrieve the most relevant objects, and then pass them to the FriendliAI generative model to generate outputs.
+
+![RAG integration illustration](../_includes/integration_friendliai_rag.png)
+
+## Requirements
+
+### Weaviate configuration
+
+Your Weaviate instance must be configured with the FriendliAI generative AI integration (`generative-friendliai`) module.
+
+<details>
+  <summary>For Weaviate Cloud (WCD) users</summary>
+
+This integration is enabled by default on Weaviate Cloud (WCD) serverless instances.
+
+</details>
+
+<details>
+  <summary>For self-hosted users</summary>
+
+- Check the [cluster metadata](../../config-refs/meta.md) to verify if the module is enabled.
+- Follow the [how-to configure modules](../../configuration/modules.md) guide to enable the module in Weaviate.
+
+</details>
+
+### API credentials
+
+You must provide a valid FriendliAI API key to Weaviate for this integration. Go to [FriendliAI](https://docs.friendli.ai/openapi/create-chat-completions) to sign up and obtain an API key.
```

Comment:
Is it valid link?

Reply:
But description says "go to ... sign up and obtain an API key". Isn't it should be `suite.friendi.ai`?

Reply:
@snowman39